de.fu_berlin.inf.dpp.core/test/junit/de/fu_berlin/inf/dpp/git/JGitFacadeTest.java

```diff
@@ -0,0 +1,160 @@
+package de.fu_berlin.inf.dpp.git;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.io.IOException;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.api.errors.InvalidRemoteException;
+import org.eclipse.jgit.api.errors.TransportException;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class JGitFacadeTest {
+
+  @Rule public TemporaryFolder tempFolder = new TemporaryFolder();
+
+  // the local user is creating the bundle
+  private File localWorkDir;
+  // the remote user is receiving the bundle
+  private File remoteWorkDir;
+  // empty directory
+  private File emptyDir;
+  // testfile
+  private File testFile;
+
+  @Before
+  public void SetUp() {
+    try {
```

Comment:
Please add only code into the setUp method which is used by each test. Create a private setup method (which is called by the corresponding tests) if you need a specific setup for a sub-set of tests.